Mathlib/Algebra/Quaternion.lean

```diff
@@ -55,8 +55,11 @@ Implemented as a structure with four fields: `re`, `imI`, `imJ`, and `imK`. -/
 structure QuaternionAlgebra (R : Type*) (a b : R) where
   /-- Real part of a quaternion. -/
   re : R
+  /-- First imaginary part of a quaternion (i). -/
```

Comment:
I could also imagine this phrasing (same for the other three); this parses better for me:
```suggestion
  /-- First imaginary part (i) of a quaternion. -/